docs/css/scaffolding.md

```diff
@@ -88,7 +91,7 @@ The simple-vars plugin exposes a variables property, in order to inject variable
 }
 ```
 
-You can work providing a [configuration file](https://github.com/leroy-merlin-br/garden/blob/master/src/css/defaults.json) with your own defaults, or override during compilation time:
+To set your own default values, you can provide a [configuration file](https://github.com/leroy-merlin-br/garden/blob/master/src/css/defaults.json), or override variables during compilation time:
```

Comment:
What do you think about `override the variables`  instead of  `override variables` ?